module-2/backend/week_one.md

```diff
@@ -18,15 +27,38 @@ Note: When you're done, submit a PR.
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
+ - between do and end add @name = 'Mr. Ed'

```

Comment:
using `@` makes the variable an instance variable. what if we want to send a local variable?